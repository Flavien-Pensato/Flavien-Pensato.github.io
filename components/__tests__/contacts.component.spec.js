import React from 'react'
import { shallow } from 'enzyme'
import serializer from 'jest-emotion'

import { Contacts } from '../contacts.component'

expect.addSnapshotSerializer(serializer)

jest.unmock('../contacts.component.js')

describe('<Contacts />', () => {
  const getShallow = () => shallow(<Contacts />)

  it('should match snapshot', () => {
    expect(getShallow()).toMatchSnapshot()
  })
})
