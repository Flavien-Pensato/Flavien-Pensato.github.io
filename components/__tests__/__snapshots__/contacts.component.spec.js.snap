// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Contacts /> should match snapshot 1`] = `
<Styled(footer)>
  <OutboundLink
    eventLabel="Github"
    onClick={null}
    target="_blank"
    title="Github link"
    to="https://github.com/Flavien-Pensato"
    trackerNames={null}
  >
    <i
      className="nes-icon github is-medium"
    />
  </OutboundLink>
  <OutboundLink
    eventLabel="Linkedin"
    onClick={null}
    target="_blank"
    title="Linkedin link"
    to="https://www.linkedin.com/in/flavien-pensato-708190a7"
    trackerNames={null}
  >
    <i
      className="nes-icon linkedin is-medium"
    />
  </OutboundLink>
</Styled(footer)>
`;
